db/migrate/20211203194425_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] The reference is not directly `user`, it is instead `author` to table `user`. Kindly see the below example on how to make such migration for adding reference with a different name than the reference model class name and make changes in your database. :pray: 

![image](https://user-images.githubusercontent.com/64894927/144670150-16acdab0-058f-47dd-a019-f89c3dd7d36e.png)